microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -424,21 +484,32 @@ In the **Elements** tool, in the **Styles** tab, click on any color preview.  Th
 
 <!-- https://developer.chrome.com/blog/new-in-devtools-102/#color-picker -->
 
-*  This checkbox is not present in Microsoft Edge Stable v101.
-*  This checkbox is present in Microsoft Edge Canary v104.
+*  This checkbox is present in Microsoft Edge Stable v102.
+*  This checkbox is present in Microsoft Edge Canary v105.
 
 See also:
 * [Change colors with the Color Picker](../css/reference.md#change-colors-with-the-color-picker) in _CSS features reference_
 
 
+<!-- ====================================================================== -->
+## Group sources into Authored and Deployed trees
```

Comment:
@captainbrosset @zoherghadyali - need text for v105 Experiments checkbox "Group sources into Authored and Deployed trees"